controllers/openstackclient_controller.go

```diff
@@ -122,15 +122,26 @@ func (r *OpenStackClientReconciler) Reconcile(req ctrl.Request) (ctrl.Result, er
 	}
 
 	// for now there is only support for a single openstackclient container per cr
-	hostKey := fmt.Sprintf("%s-%d", instance.Name, 0)
-	hostname, err := common.CreateOrGetHostname(instance, hostKey, openstackclient.Role)
+	hostnameDetails := common.Hostname{
+		IDKey:    fmt.Sprintf("%s-%d", strings.ToLower(openstackclient.Role), 0),
+		Basename: openstackclient.Role,
+		VIP:      false,
+	}
+	err = common.CreateOrGetHostname(instance, &hostnameDetails)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	r.Log.Info(fmt.Sprintf("OpenStackClient hostname set to %s", hostname))
+	r.Log.Info(fmt.Sprintf("OpenStackClient %s hostname set to %s", hostnameDetails.IDKey, hostnameDetails.Hostname))
+
+	// TODO: maschuppert - atm the openstackclient only has the ctlplane network.
+	//       for debugging it probably makes sense to attach the openstackclient to all networks?
+
+	// update VIP network status
```

Comment:
Is this actually VIP network status, given `VIP: false` on line 128?

Reply:
yes,  it is not. previously I had openstackclient VIP: true as there might be always just a single openstackclient, but switched it to false later and missed to update the comment here, thanks!

Reply:
updated